Packs/Base/TestPlaybooks/playbook-Onion_address_extraction_test.yml

```
@@ -0,0 +1,457 @@
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "11"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: invalid_addresses
+      value:
+        simple: '"abcdefghijkmnop.onion", "pqrstuvwx2yzabcdeghijklmnopqrstufffwx2yzabcde2jklmnopqw.onion"'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 1360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 0d20e8b1-afb6-4a1a-81f7-9e25e000e267
+    type: title
+    task:
+      id: 0d20e8b1-afb6-4a1a-81f7-9e25e000e267
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 1710
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: 4355663b-cd5b-45b9-8b99-6e8da24e67b2
+    type: regular
+    task:
+      id: 4355663b-cd5b-45b9-8b99-6e8da24e67b2
+      version: -1
+      name: DeleteContext
+      description: |-
+        Delete field from context.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: DeleteContext
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      all:
+        simple: "yes"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 1185
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 354b7c68-4cc4-4722-8d91-a55ebbc63072
+    type: condition
+    task:
+      id: 354b7c68-4cc4-4722-8d91-a55ebbc63072
+      version: -1
+      name: Check that no invalid indicators were extracted
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "9"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotExists
+          left:
+            value:
+              simple: ExtractedIndicators
```

ExtractedIndicators.Onion_Address
```suggestion
              simple: ExtractedIndicators
```

Should be more accurate and dedicated for that specific type you're checking

---

Im not checking Onion addresses here, im checking that no other indicatro type was created by mistake.